.prettierrc

```
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 4,
+  "semi": true,
+  "singleQuote": true,
+  "useTabs": true
```

I just peeked through https://prettier.io/docs/en/options.html and confirmed:
* [semicolons are enabled by default](https://prettier.io/docs/en/options.html#semicolons)
* [trailing commas are `es5` by default](https://prettier.io/docs/en/options.html#trailing-commas)

...so, really, the only overridden setting we need to specify here is `"useTabs": true`.

```suggestion
  "useTabs": true
```